src/timeMachine/components/Vis.tsx

```diff
@@ -84,12 +84,12 @@ const TimeMachineVis: FC<Props> = ({
 
   // Handles deadman check edge case to allow non-numeric values
   const viewRawData =
-    isViewingRawData ||
+    isViewingRawData &&
```

Comment:
Taking this entire block out resulted in failed checks e2e tests, but leaving it as OR caused everything to show as the old raw table view.  This seemed simple enough and got tests passing locally but if there's a better way here I'm all for it.

Reply:
ugh this logic will forever haunt me. I think your change looks good tho

Reply:
ah wait. It might not actually work. I think this would only show the raw data view for numeric checks. If you run a query that returns string results, it should render the raw data view automatically. Does that still work? I'm also a little confused why we have to touch this logic at all just to remove this flag 🤔

Reply:
Ended up reverting this change.